test/integration/targets/aws_codebuild/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+---
+# tasks file for aws_codebuild
+
+- name: Run aws_codebuild integration tests.
+
+  block:
+
+    # ==================== preparaions ========================================
+
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: yes
+
+    - name: create IAM role needed for CodeBuild
+      iam_role:
+        name: "{{ resource_prefix }}-codebuild-service-role"
```

Comment:
The CI testrunner fails to create this IAM role (AccessDenied), so what do I need to do to get this working?

Reply:
We'll need to update our CI system to accommodate the additional permissions required.